tests/general/InvalidLinkBearTest.py

```diff
@@ -82,6 +82,8 @@ def test_run(self):
         valid_file = """
         http://httpbin.org/status/200
         http://httpbin.org/status/201
+        http://httpbin.org/status%20is/200
+        http://www.google.com/hello%20world/200
```

Comment:
Which line specifically ? In the review, you seem to have highlighted both the lines I have added i.e. ``http://httpbin.org/status%20is/200`` and ``http://www.google.com/hello%20world/200``.
Ok, adding a test for the invalid URLs as well :)